pages/mesosphere/dcos/2.2/release-notes/2.2/index.md

```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
 
 # Custom CA certificate rotation
-DC/OS now allows an operator to rotate the custom CA certificates by simply updating configuration settings during an upgrade. This feature ensures all the services that are using the custom CA based certificates are automatically updated after an upgrade.
+DC/OS now allows an operator to rotate the custom CA certificates by simply updating configuration settings during an upgrade. This feature ensures all the services that are using the custom CA based certificates are automatically updated after an upgrade. 
 
 # Breaking changes
 
 # Component Versions
 DC/OS 2.2.0 includes the following component version updates:
 
+- Marathon 1.11.16
+
 # Fixed and Improved Issues
+DC/OS 2.2 fixes the following issues:
+
+- systemd errors were being thrown during patch upgrades. (COPS-6506)
+- dcos-fluent-bit.service was consuming too much memory. (COPS-6218) 
+- Selecting *Install DC/OS CLI* presents a URL to a binary that is incorrect. (COPS-6360)
+- Mesos Authorizer module was trying to authorize the dcos_anonymous account on permissive security mode. (COPS-6335, D2IQ-70037) 
+- Tasks would not start until Telegraf created /run/dcos/telegraf/dcos_statsd.sock. (COPS-6355)
+- DC/OS installations on Flatcar Linux would not finish due to Java processing issues. (COPS-6422, D2IQ-70809, COPS-6190) 
+- Unmounting issues in Mesos external persistent volumes. (COPS-5920)
+- RootCA bundle was not written soon enough for Docker to find it before it exhausts startup limits on a reboot. (COPS-6534)
+- UCR fetcher had issues pulling down images. (COPS-6381) 
+- dcos-net was not always able to set all Spartan IP addresses after node reboot. (COPS-6519)
+- dcos-telegraf.socket was down after a patch. (COPS-6512)
+- Marathon migration failed while upgrading from 1.13.7 to 2.0.2. (COPS-5897)
+- Nested Marathon groups were not inheriting enforceRole behavior from top-level groups. (COPS-6529)
+- enforceRole was automatically being set to false on all Marathon groups. (COPS-6533)
+- Selecting *Run* on a job or selecting *Delete* to remove a group had no effect and threw an exception. (COPS-6324) 
+- etcd provided an incorrect response in calicoctl. (COPS-6341)  
+- Users were unable to remove empty folders from Metronome. (COPS-6139) 
+- Hostname Resolution was failing for VIPs. (COPS-6411)  
+- Using file-based secrets caused mount failure and issues in the json editor. (COPS-6085, D2IQ-68114)  
+- dse-0-node failed with ssl verification during Datastax-DSE upgrade from 2.1.2-5.1.2 to 2.2.0-5.1.2 on DC/OS 2.0.2. (COPS-6116)
+- CRDB was running and bouncer can not connect to it, because the bouncer's SELECT query fails. (COPS-5979) 
+- An unknown response code was received when querying a DC/OS endpoint. (COPS-5915, D2IQ-65296)  
```

Comment:
Delete this one and add the COPS/D2iQ numbers to the previous item as they are related:
```suggestion
```